tools/metrics/grafana/grafana.go

```diff
@@ -174,9 +174,13 @@ func exportNormalizedDashboard(d *Dashboard) error {
 		return err
 	}
 	outPath := filepath.Join(dashboardsDir, fileName)
+	outFile, err := os.ReadFile(outPath)
+	if err != nil {
```

Comment:
nit: the original intent was to write the file if it did not exist or if it contains different bytes

```suggestion
	if err != nil && !os.IsNotExist(err) {
```

Reply:
In case the file does not exist, the byte slice content return will be nil and after the `bytes.Equal` result 🤔 

I guess we want to write when the previous file was not there (i.e. on the first time exporting)?

Reply:
I sent a fix that would handle the nil content case.